File: hw/basurto/u3/hw23AbstractFactory/hw23AbstractFactory.java/AbstractFactory/src/main/java/ec/edu/espe/AbstractFactory/model/WinFactory.java

package ec.edu.espe.AbstractFactory.model;

/**
 *
 * @author Esteban Basurto,CodeBreakers,@ESPE
 */
public class WinFactory extends GUIFactory {
    @Override
    public Button createButton() {
        return new WinButton();
    }
    @Override
    public Menu createMenu() {
        return new WinMenu();
    }
}

File: hw/chavez/u3/hw24AbstractFactory/JavaScript/model/WinFactory.js
const GUIFactory = require("./GUIFactory");
const WinButton = require("./WinButton");
const WinMenu = require("./WinMenu");

class WinFactory extends GUIFactory {

    createButton() {
        return new WinButton();
    }

    createMenu() {
        return new WinMenu();
    }
}

module.exports = WinFactory;